inc/template-functions.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * Additional features to allow styling of the templates
+ * Functions which enhance the content displayed by hooking into WP
```

Comment:
Let's not abbreviate WordPress here, just so it's clear. Also, maybe change "the content" to "the theme" since what's changed could extend beyond actual content.